src/lib/api.ts

```diff
@@ -38,22 +38,30 @@ export async function getCollections(api) {
   );
   const relations = relationsRes.data.data;
   relations.forEach((relation) => {
-    const oneField = collections[relation.meta.one_collection]?.fields.find(
```

Comment:
my bug was `relation.meta.one_collection` being null, so this extension wouldn't work.